### ISSUE TITLE
"Restrict the use of certain cryptographic algorithms" -- windows version applies to?

### ISSUE THREAD

**Issue by repton-infinity** [MicrosoftDocs/SupportArticles-docs#61]
Path: support/windows-server/windows-security/restrict-cryptographic-algorithms-protocols-schannel.md
available as: https://docs.microsoft.com/en-US/troubleshoot/windows-server/windows-security/restrict-cryptographic-algorithms-protocols-schannel?source=docs

At the top it says:

"Original product version:   Windows Server 2012 R2"

Just below, it says:

"This article applies to Windows Server 2003 and earlier versions of Windows. For registry keys that apply to Windows Server 2008 and later versions of Windows, see the TLS Registry Settings."

When I click on "TLS Registry Settings", it takes me to this page: https://docs.microsoft.com/en-us/windows-server/security/tls/tls-registry-settings
At the top of this page, it says:

"Applies to: Windows Server (Semi-Annual Channel), Windows Server 2019, Windows Server 2016, Windows 10"

----

Basically I'm just trying to figure out how to make a change on 2012 R2, and it's really unclear which page I should read.

**Comment by Deland-Han** [comment]
@repton-infinity, thank you for your feedbacks. I have changed the applies to line of this article to Windows Server 2003.
